app/views/includes/nhs.uk/footer.html

```diff
@@ -1,7 +1,18 @@
 <footer id="footer" role="contentinfo">
 
   <div class="footer-wrapper">
-    
+
+    <div class="text">
+
+      <p>
+        <small>
+          &copy; Crown copyright<br>
+          Content available under <a href="#">Open Government Licence v3.0</a>, except where otherwise stated.
```

Comment:
Can we link to the actual license here? https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/